publish-all-npm-packages.sh

```diff
@@ -6,10 +6,10 @@ set -x
 export PACKAGES="aws-cache-helpers"
 for package in $PACKAGES
 do
-  echo "Publishing packages with version $MOMENTO_CODE_REVISION_ID"
+  echo "Publishing packages with version $VERSION"
   pushd $package
     mv package.json package.json.ORIG
-    cat package.json.ORIG|jq ". += {\"version\": \"$MOMENTO_CODE_REVISION_ID\"}" > package.json
+    cat package.json.ORIG|jq ". += {\"version\": \"$VERSION\"}" > package.json
```

Comment:
I don't think we own the namespace `@gomomento-poc` yet, so I don't think publish rn will work. We can try, but I think we need to setup owner ship of this namespace, and possible create the package first on npmjs.org before we can publish it

https://github.com/momentohq/ts-pocs/blob/main/aws-cache-helpers/package.json#L2

Reply:
Let me merge this and will do follow up PR make new namespace if thats the case interested to see how it fails.